modules/notifications/notifications.php

```diff
@@ -550,10 +550,14 @@ function notification_status_change( $new_status, $old_status, $post ) {
 			
 			/* translators: 1: date, 2: time, 3: timezone */
 			$body .= sprintf( __( 'This action was taken on %1$s at %2$s %3$s', 'edit-flow' ), date_i18n( get_option( 'date_format' ) ), date_i18n( get_option( 'time_format' ) ), get_option( 'timezone_string' ) ) . "\r\n";
-			
-			$old_status_friendly_name = $this->get_post_status_friendly_name( $old_status );
-			$new_status_friendly_name = $this->get_post_status_friendly_name( $new_status );
-			
+
+			if ( $old_status == 'new' ) {
+				return; 
```

Comment:
Instead of returning here if the old status is 'new' we should still allow a notification to be sent. We have already created a subject to handle this situation in line 514.

If we were to supply an `$old_status_friendly_name` for this instance it would prevent the call to `get_post_status_object` in line 557 failing and causing an error for new posts.